lib/std/json.zig

```diff
@@ -2717,7 +2721,7 @@ test "encodesTo" {
     try testing.expectEqual(true, encodesTo("withąunicode😂", "with\\u0105unicode\\ud83d\\ude02"));
 }
 
-test "issue 14600" {
+test "deserializing string with escape sequence into sentinel slice" {
```

Comment:
This is just a small thing part of the effort to decouple the codebase from GitHub which IIUC is a goal.